components/builder-web/app/origin/origin-page/origin-jobs-tab/_origin-jobs-tab.component.scss

```diff
@@ -0,0 +1,8 @@
+.origin-jobs-tab-component {
+  display: flex;
+  flex-flow: column;
+
+  > .jobs-list {
+    width: 100%;
```

Comment:
This has no effect, as the element is currently `display: inline`. We can just remove it.

Reply:
✔️